### DISCUSSION TITLE
Get User Connections

### DISCUSSION THREAD

**Discussion by michel929** [discord-jda/JDA#2374] [Questions and Help]
Is there a way to get the connections from other accounts, such as the SteamID or Username?

**Comment by ghost** [comment]
not with jda
you can use `https://discord.com/api/v9/users/ID/profile` endpoint and get the connected accounts json array
the array will have a list of a buncha json objects that show the connection type and its id

**Reply by Wibrr** [reply]
{"message": "401: Unauthorized", "code": 0}
You sure?
I am logged into discord and the discord developer portal and it doesn't seem to work, even after putting in my own ID in the URL.

**Comment by srnyx** [comment]
It's technically possible (I think), just doesn't exist in JDA I guess: https://discord.com/developers/docs/resources/user#connection-object

**Comment by burdoto** [comment]
hi can we eventually get JDA to cover this functionality, as it becomes more and more relevant, and i would like to avoid writing some middleman class
